HopsanGUI/LogVariable.h

```diff
@@ -379,5 +376,17 @@ SharedVectorVariableT switchVariableGeneration(SharedVectorVariableT pVar, int g
 double pokeVariable(SharedVectorVariableT a, const int index, const double value);
 double peekVariable(SharedVectorVariableT a, const int b);
 
+template<typename Function>
+QVector<double> invokeMathFunctionOnData(VectorVariable *pVector, Function func)
```

Comment:
I am not sure about this idea, but I think this one could have been a template member function in the VectorVariable class, then there would not have been a need to expose the getCachedDataVector(), but not so important.

Reply:
Good point. I moved `invokeMathFunctionOnData` to a member function and removed `getCachedDataVector`.